.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 120,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "semi": true,
+  "tabWidth": 2,
+  "bracketSpacing": false
+}
```

Comment:
Any reasons for this specific configuration? I would prefer to use the configuration as defined by Prettier RFC: https://emberjs.github.io/rfcs/0628-prettier.html#prettier Mostly to use the same as other Ember addons.